central/metrics/central.go

```diff
@@ -178,6 +178,13 @@ var (
 		Name:      "cluster_metrics_cpu_capacity",
 		Help:      "Total Kubernetes cpu capacity of all nodes in a secured cluster",
 	}, []string{"ClusterID"})
+
+	totalOrphanedPLOPCounter = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Namespace: metrics.PrometheusNamespace,
+		Subsystem: metrics.CentralSubsystem.String(),
+		Name:      "total_orphaned_plop_counter",
```

Comment:
```suggestion
		Name:      "orphaned_plop_total",
```
I think this would be more in line with common Prometheus naming conventions. Counters usually add in `_total` and don't have the word counter in them.